### DISCUSSION TITLE
Marine Forecast Layers

### DISCUSSION THREAD

**Discussion by isingh** [weather-gov/api#725] [General]
It seems the marine forecast API is getting [EOL](https://www.weather.gov/media/notification/pdf_2023_24/scn24-50_removal_of_digital_projects_from_forecast_aaa.pdf). 

Urls of the format: `https://m1.weather.gov/MapClick.php?FcstType=digitalJSON&lat=32.7554&lon=-117.2576` return marine forecasts like so:

```
"LateAfternoon": {
    "periodName": "Late Afternoon",
    "time": [
      "4 pm",
      "5 pm"
    ],
    "unixtime": [
      "1717542000",
      "1717545600"
    ],
    "windSpeed": [
      "7",
      "7"
    ],
    "windGust": [
      "null",
      "null"
    ],
    "significantWaveHeight": [
      "3",
      "3"
    ],
    "swellHeight": [
      "3",
      "3"
    ],
    "swellDirection": [
      "270",
      "250"
    ],
    "swellDirectionCardinal": [
      "W",
      "WSW"
    ],
    "swellPeriod": [
      " 8",
      " 7"
    ],
    "swell2Height": [
      "2",
      "2"
    ],
    "swell2Direction": [
      "230",
      "240"
    ],
    "swell2DirectionCardinal": [
      "W",
      "WSW"
    ],
    "swell2Period": [
      "16",
      "15"
    ],
    "windWaveHeight": [
      "1",
      "1"
    ],
    "windDirectionCardinal": [
      "SSW",
      "SSW"
    ],
    "windDirection": [
      "210",
      "210"
    ]
  },
```

While this should have moved to the hourly forecast API, I don't see this data. For example, the gridpoint for the same coordinates is `https://api.weather.gov/gridpoints/SGX/54,17/forecast/hourly?units=si` which doesn't have any such layers. I tried a few gridpoints close to the lat/lon but same results. I understand from OpenAPI definition that those layers are optional, but I cant seem to find that data in any grid point close to the coast. Any ideas on how I can retrieve this data? 

This was discussed [earlier](https://github.com/weather-gov/api/discussions/662#discussioncomment-7198850) and it seems the suggestion was to stick with forecast API?

**Comment by sullynole** [comment]
The raw data endpoint should have everything there, but format is slightly different. https://api.weather.gov/gridpoints/SGX/53,16

**Reply by isingh** [reply]
Thank you! Appreciate the correction. 

Btw, I realized that even subtle shifts in lat/lon give the grid as SGX/53,17 which has a weird almost all 0 data for waves.
